tests/test.py

```diff
@@ -40,21 +40,34 @@ class Event(TypedDict, total=False):
 TESTDIR = Path(__file__).parent.resolve()
 ROOTDIR = TESTDIR.parent.resolve()
 
-exe = subprocess.check_call
-
 # check if root fs is btrfs
 root_is_btrfs = subprocess.check_output(["findmnt", "--noheadings", "--output=FSTYPE", "/"]).strip() == b"btrfs"
 
-
-def slow_exe(argv: list[str], **kwargs) -> None:
+_exe_stream = None
+def _exe(argv_any: tuple[Any, ...], **kwargs) -> None:
+    argv: list[str] = [str(x) for x in argv_any]
+    print(f"Running command: {' '.join(argv)}", file=_exe_stream)
+    proc = subprocess.run(argv, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False, **kwargs)
```

Comment:
This is unsafe as a generic API. If a process outputs too much stuff on stdout and nobody reads the buffers, it'll block. The previous code used `stdout=NULL` for that reason.